common/components/WalletDecrypt/index.tsx

```diff
@@ -24,6 +25,8 @@ import { AppState } from 'reducers';
 import Web3Decrypt from './Web3';
 import Help from 'components/ui/Help';
 import { knowledgeBaseURL } from 'config/data';
+import NavigationPrompt from 'containers/Tabs/SendTransaction/components/NavigationPrompt';
```

Comment:
Consider moving `NavigationPrompt` into `components`, as its really not specific to `SendTransaction` anymore.